pkg/daemon/controller/remotevtep.go

```diff
@@ -0,0 +1,64 @@
+package controller
+
+import (
+	"fmt"
+	"net"
+	"sync"
+
+	ramav1 "github.com/oecp/rama/pkg/apis/networking/v1"
+)
+
+type RemoteVtepCache struct {
```

Comment:
Actually, this cache is for node ip resolving, and not just for vteps. Maybe reuse the NodeIPCache is better.